noggin/form/edit_user.py

```diff
@@ -111,6 +111,12 @@ class UserSettingsAddOTPForm(ModestForm):
         description=_("please reauthenticate so we know it is you"),
     )
 
+    otp = StringField(
+        _('One-Time Password'),
+        validators=[Optional()],
+        description=_("Enter your One-Time Password token if you have enrolled one"),
```

Comment:
Actually, the field will only be displayed if there is an existing token, so you don't need "if you have enrolled one".